README.md

```diff
@@ -116,25 +110,34 @@ In a different example, a service using the Functions Framework could have the u
 
 - The steps taken by the framework in response to ingress requests
 - The function signatures which developers must adhere to when writing functions for use with a Functions Framework
 - The mechanism by which the developer signals that the function has completed performing useful work
 
 The developer must signal the function's signature type to the framework. This enables the framework to take appropriate unmarshalling, invocation and completion signalling steps.
 
-### HTTP Functions
+### HTTP Functions (Signature Type: `http`)
+
+The Functions Framework **must** support signature type `http`.
 
 When the container receives an ingress request, the framework must invoke the developer's function by passing a language-idiomatic representation of HTTP objects as arguments to the function. These objects must enable the developer to perform common HTTP tasks, such as inspecting the request's content encoding or headers. These objects should be accurate representations of the HTTP request received by the execution environment (i.e., path, body and headers should not be modified before passing them to the user's function).
 
-The developer's function must explicitly signal that it has completed performing useful work by sending a response to the Functions Framework. This response signals that the stateless container can be stopped.
+### CloudEvents Functions (Signature Type: `cloudevents`)
 
-### CloudEvents Functions
+The Functions Framework **must** support signature type `cloudevents`.
 
 When the container receives an ingress request, the framework must invoke the developer's function by passing an object corresponding to a [CloudEvents type](https://github.com/cloudevents/spec/blob/master/spec.md). This object does not expose HTTP semantics to the developer's function. The framework must handle unmarshalling HTTP requests into the CloudEvents object that is passed to the developer's function, and should support both [binary and structured content modes](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#3-http-message-mapping) for incoming HTTP CloudEvent requests.
 
-The developer's function must either explicitly or implicitly signal that it has completed performing useful work. The function may explicitly signal this condition by explicitly returning. The function may implicitly signal this condition by simply evaluating until it reaches the end of the function's code block.
+### Legacy Events (Signature Type: `event`)
+
+The Functions Framework **may** support signature type `events`. This signature supports non-CloudEvent style events. Your function have must use the following signature:
+
+- 1st parameter `data`
+- 2nd parameter `context`
```

Comment:
I like the clarity of showing a function signature outside a sentence/prose structure. Can we get this for the other signature types?

Reply:
Added for `cloudevent` type.